cranelift/codegen/src/isa/x64/abi.rs

```diff
@@ -520,3 +551,482 @@ impl ABIBody for X64ABIBody {
         unimplemented!()
     }
 }
+
+// TODO refactor
+/// Returns the size (in bits) of a given type.
+pub fn ty_bits(ty: Type) -> usize {
+    match ty {
+        B1 => 1,
+        B8 | I8 => 8,
+        B16 | I16 => 16,
+        B32 | I32 | F32 => 32,
+        B64 | I64 | F64 => 64,
+        B128 | I128 => 128,
+        IFLAGS | FFLAGS => 32,
+        B8X8 | I8X8 | B16X4 | I16X4 | B32X2 | I32X2 => 64,
+        B8X16 | I8X16 | B16X8 | I16X8 | B32X4 | I32X4 | B64X2 | I64X2 => 128,
+        _ => panic!("ty_bits() on unknown type: {:?}", ty),
+    }
```

Comment:
```suggestion
    ty.bits()
```

Reply:
Thanks! Even updated the callers so they use ty.bits() instead.